test/indexer/test_hca_indexer.py

```diff
@@ -1335,6 +1335,87 @@ def test_multiple_samples(self):
                 for sample in contents['samples']:
                     self.assertIn(sample['entity_type'], sample_entity_types)
 
+    def test_sample_with_no_donor(self):
+        """
+        Index two bundles for the same project, one bundle has a sample that
+        is connected to a donor and the other bundle has a sample that is not.
+        Verify the lack of a donor is represented in the project aggregate.
+        """
+        # Sample (Specimen): 70d2b85a, Donor: b111e5bf
+        bundle_fqid = self.bundle_fqid(uuid='1fd499c5-f397-4bff-9af0-eb42c37d5fbe',
+                                       version='2021-03-18T11:38:49.884Z')
+        self._index_canned_bundle(bundle_fqid)
+        # Sample (Organoid): df23c109, Donor: none
+        bundle_fqid = self.bundle_fqid(uuid='0722b70c-6778-423d-8fe9-869e2a515d35',
+                                       version='2021-03-18T11:38:49.863Z')
+        self._index_canned_bundle(bundle_fqid)
+        donor = [
```

Comment:
There is some plural vs singular confusion going on. If the value is a list a more appropriate name would be `donors`. Then again this is unpacked usign `one` twice but the list is only needed once.